openforcefield/tests/test_molecule.py

```diff
@@ -188,27 +247,27 @@ def test_add_conformers(self):
         conf_missing_z = unit.Quantity(np.array([[101., 102. ,103.], [104. ,105. ,106.], [107., 108., 109.],
                                         [110.,111.,112.],   [113.,114.]]),
                                         unit.angstrom)
-        with self.assertRaises(Exception) as context:
+        with pytest.raises(Exception) as excinfo:
```

Comment:
Agree with putting in `pytest.raises`. For some reason I don't remember, I usually have `context` where you put `excinfo`. I've never actually done anything with the variable, but I'd be happy to switch to `excinfo` if that's a more informative name for the data stored there. What do you think?

Reply:
I shamelessly speculated that `context` was the UnitTest terminology so I changed it for the syntax suggested in the `pytest` example here: https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions . I'll be happy with any convention though!

Reply:
Gotcha. I'll probably start using `excinfo` from now on.